android/modules/media/src/java/ti/modules/titanium/media/TiCameraActivity.java

```diff
@@ -161,6 +164,13 @@ public void onCreate(Bundle savedInstanceState)
 
 	}
 
+	public void setFullscreenForCamera(boolean fullscreen)
```

Comment:
checking for truth value is not required since we never set back from fullscreen.